ui/app/templates/components/job-editor.hbs

```diff
@@ -110,7 +88,7 @@
   </div>
   <div
     class="boxed-section
-      {{if this.planOutput.failedTGAllocs 'is-warning' 'is-primary'}}"
+      {{if this.planOutput.failedTGAllocs 'is-warning'}}"
```

Comment:
Breaks `a11y` color rule.

Reply:
Okay; let's work with design to find something that indicates a success. We'll have this situation come up in the future in other contexts as well, I suspect, and worth setting a good precedent here.